lib/venue.rb

```diff
@@ -1,2 +1,30 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons
+
+  def initialize(name, capacity, patrons = [])
+    @name = name
+    @capacity = capacity
+    @patrons = patrons
+  end
+
+  def add_patron()
+    @patrons['name']
```

Comment:
`@patrons` is an array, so how could you add `new_patron` to the array? If you're unsure, try checking the exploring #each lesson.